department-of-veterans-affairs/va.gov-cms
# VACMS-4932: Add VAMC menu block to VAMC node view (proofing) pages.

## Description
Closes #4932

## Testing done
Visual / behat

## QA steps
This will require a bunch of clicking around / spot checking, e.g: Visit multiple VAMC's and subordinate pages in said VAMC's, confirming that the contextually appropriate menu appears (just once) on each page. We will also need to confirm that VAMC menus do not appear in places where they shouldn't, i.e.: admin pages, user pages, node edit pages, etc.

Check that the correct VAMC menu is there

- [ ] 1. On VAMC facility nodes 
  - [ ] Go to /admin/content and filter VAMC facility
  - [ ] Open a variety of recently updated facilities to see if the menu is there, and that it's the right menu
- [ ] 2. On events
  - [ ] Go to /admin/content and filter by Event nodes
  - [ ] Open a variety of recently updated events with a section set to VAMC, to see if the menu is there, and that it's the right menu
- [ ] 3. On Policies pages
  - [ ] Go to /admin/content and filter by nodes with "Policies" in the title
  - [ ] Open a variety of recently updated nodes with a section set to VAMC, to see if the menu is there, and that it's the right menu



### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
